plugins/org.eclipse.xtend.core/build.properties

```diff
@@ -12,6 +12,8 @@ bin.includes = META-INF/,\
 exclude.. = **/GenerateXtend.mwe2
 src.includes = about.html
 additional.bundles = org.eclipse.xtext.generator,\
-                     org.eclipse.xtext.xtext.generator
+                     org.eclipse.xtext.xtext.generator,\
+                     org.eclipse.emf.mwe2.launch,\
```

Comment:
org.eclipse.emf.mwe2.launch is treated as required dependency.
mwe2 is a downstream project that depends on xtext and can not be consumed during xtext build.